src/cbor.zig

```diff
@@ -318,6 +335,31 @@ pub const ArrayIterator = struct {
     }
 };
 
+/// Iterator for iterating over an indefinite length array, returned by DataItem.arrayIndef()
+pub const IndefArrayIterator = struct {
+    data: []const u8,
+    i: usize,
+    count: usize,
```

Comment:
I think this can be removed. The `ArrayIterator` uses it to compare the current number of seen data items to the number of expected DIs. As we don't know the number of expected DIs, this field serves no purpose.